.travis.yml

```diff
@@ -3,7 +3,7 @@ scala:
 - 2.10.6
 - 2.12.3
 jdk:
-- oraclejdk8
+- openjdk8
```

Comment:
Oh good catch here!

Reply:
Not a good catch. It's breaking every build now. :)